src/client/fuse_ll.cc

```diff
@@ -753,6 +753,15 @@ static void fuse_ll_rename(fuse_req_t req, fuse_ino_t parent, const char *name,
 #endif
                            )
 {
+#if FUSE_VERSION >= FUSE_MAKE_VERSION(3, 0)
+  // cephfs does not support renameat2 flavors; follow same logic as done in
+  // kernel's ceph_rename()
```

Comment:
extreme nit: this part of comment can be `kclient's ceph_rename()` else LGTM

Reply:
Fair enough. Fixed.